src/main/java/uk/gov/ons/census/action/utility/FulfilmentMapper.java

```diff
@@ -0,0 +1,19 @@
+package uk.gov.ons.census.action.utility;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import org.springframework.jdbc.core.RowMapper;
+import uk.gov.ons.census.action.model.dto.FulfilmentMapperDTO;
+
+public class FulfilmentMapper implements RowMapper<FulfilmentMapperDTO> {
```

Comment:
This can be a component, and not 'newed' up each time it's needed, it can still be injected.

Reply:
What does Component do?

Reply:
Just uses Spring's standard wiring then to create a singleton version of the FulfilmnentMapper class, so you send that rather than new one up.